lib/teslamate_web/templates/car/summary.html.leex

```diff
@@ -158,7 +158,29 @@
           <%= unless is_nil(@summary.battery_level) do %>
           <tr>
             <td class="has-text-weight-medium"><%= gettext "State of Charge" %></td>
-            <td><%= @summary.battery_level %>%</td>
+            <td><%= @summary.battery_level %>% (<%= 
+ 
+                if @settings.unit_of_length == :mi do
+                  "#{Float.round(@summary.rated_battery_range_km/@summary.battery_level * 100 / 1.60934, 2)} mi"
+                else
+                  "#{Float.round(@summary.rated_battery_range_km/@summary.battery_level * 100,2)} km"
+                end
+ 
+              %> 100%)</td>
+          </tr>
+          <% end %>
+          <%= unless is_nil(@summary.usable_battery_level) do %>
```

Comment:
Does it make sense to hide the 'Usable State of Charge' row if `usable_battery_level` == `battery_level`? What do you think?

Reply:
Sure makes sense. There is no additional information when both are the same.
I'll also try to make a snowflake when difference is higher than 2% I think thats what the TeslaApp does.

Reply:
> I'll also try to make a snowflake when difference is higher than 2% I think thats what the TeslaApp does.

Yeah, that would be great!